### ISSUE TITLE
Error running timer ‘company-echo-show’

### ISSUE THREAD

**Issue by anhnhoktvn** [TommyX12/company-tabnine#9]
My config:
```
;; tabnine
(use-package company :ensure t)
(add-hook 'after-init-hook 'global-company-mode)
(use-package company-tabnine :ensure t)
(add-to-list 'company-backends #'company-tabnine)
;; Trigger completion immediately.
(setq company-idle-delay 0)

;; Number the candidates (use M-1, M-2 etc to select completions).
(setq company-show-numbers t)

;; Use the tab-and-go frontend.
;; Allows TAB to select and complete at the same time.
(company-tng-configure-default)
```

Steps to reproduce:
Just type something and an error message with popup.

```
Error running timer ‘company-echo-show’: (error "Company: backend company-tabnine error \"Invalid function: (messages (alist-get (quote promotional_message) company-tabnine--result))\" with args (meta that)")
```

**Comment by TommyX12** [comment]
Should be fixed in the latest version (by #6). Try reinstalling / updating the package.

**Comment by anhnhoktvn** [comment]
I'm using the latest version: 20181123.341
Here the function in the source code:
```
(defun company-tabnine--meta (candidate)
  "Return meta information for CANDIDATE.  Currently used to display promotional messages."
  (if (null company-tabnine--result)
      nil
    (when-let* ((messages (alist-get 'promotional_message company-tabnine--result)))
      (s-join " " messages))))

```
